README.md

```diff
@@ -59,7 +59,8 @@ JPlag is released on [Maven Central](https://search.maven.org/search?q=de.jplag)
 1. Download or clone the code from this repository.
 2. Run `mvn clean package` from the root of the repository to compile and build all submodules.
    Run `mvn clean package assembly:single` instead if you need the full jar which includes all dependencies.
-5. You will find the generated JARs in the subdirectory `cli/target`.
+   Run `mvn -Pwith-report-viewer clean package assembly:single` to build the full jar with the report viewer. In this case, you'll need `npm` installed.
```

Comment:
Should we link to nodejs and npm instead of just tagging `npm`?

Reply:
I added the link and also changed the link content text to be node.js since that is the correct term for what users need to install